ui/app/styles/components/masked-input.scss

```diff
@@ -0,0 +1,71 @@
+.masked-input {
+	position: relative;
+}
+
+.masked-input .masked-value {
+	padding-left: 2.50rem;
+}
+
+// we want to style the boxes the same everywhere so they
+// need to be the same font and small
+.masked-input.masked .masked-value {
+  font-size: 9px;
+	font-family: $family-primary;
+}
+
+.masked-input.masked .masked-value {
+  line-height: 2.5;
+}
+// aligns the boxes on the input page
+.masked-input.masked:not(.display-only) .masked-value {
+  line-height: 3;
+}
+
+//override bulma's pre styling
+.masked-input .display-only {
+	line-height: 1.5;
+	font-size: 1rem;
+}
+
+.masked-input-toggle {
+	background: transparent;
+	position: absolute;
+	height: auto;
+	top: $size-6/4;
+	bottom: $size-6/4;
```

Comment:
Looking forward to the new spacing variables :\

Reply:
Yeah 😬